CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
- 3.2.0 (2021-11-23)
+4.0.0 (2022-03-31)
```

Comment:
I managed to forget to push https://github.com/matrix-org/matrix-appservice-bridge/commit/30f6e3a4f4c8e3ec0c4d08055e39845a71d34763 until after I submitted this PR, which is why the diff looks odd. This is actually on `develop` and GitHub is just confused.